src/components/ha-sidebar.js

```diff
@@ -226,6 +226,10 @@ class HaSidebar extends
       checks--;
     } while (checks > 0 && target !== null && !attr);
 
+    if (attr === 'states') {
```

Comment:
This is not right. The sidebar shouldn't know about local storage.

Reply:
Then what do you suggest